cfgov/jinja2/v1/owning-a-home/closing-disclosure/index.html

```diff
@@ -15,34 +21,13 @@
 {% endblock %}
 
 {% block css %}
+    {{ super() }}
     <link rel="stylesheet"
           href="{{ static('apps/owning-a-home/css/main.css') }}">
-    {{ super() }}
 {% endblock css %}
 
-{% block javascript %}
-    {{ super() }}
-     <script async>
-      if ( document.body.parentElement.className.indexOf( 'no-js' ) === -1 ) {
-        !function(){
-          {# Include site-wide JavaScript. #}
-          var s = [
-            '{{ static('apps/owning-a-home/js/common.js') }}',
-            '{{ static('apps/owning-a-home/js/form-explainer/index.js') }}'
-          ];
-          jsl(s);
-        }()
-      }
-    </script>
-{% endblock javascript %}
-
 {% block content %}
```

Comment:
Because of the switch to extending `content-base.html`, you'll want to do this (and remove the `{{ super() }}` below, which I can't do in the same suggestion) to keep the block logic working as expected. With this method, it `super`s everything from `content-base`'s `content` block above it, meaning that if you put something in `post_content`, it would appear above the contents of this here `content` block.

A visualization might help:

![Using the code in this branch at the current time, when putting the text 'post_content' into the block of the same name in content-base, that text is output above the main content of the page](https://user-images.githubusercontent.com/1044670/118562972-ba363400-b73b-11eb-8e5d-06156004979d.png)

```suggestion
{% block body_content %}
```

Same comment applies to the other two files.

Reply:
Updated in https://github.com/cfpb/consumerfinance.gov/pull/6446/commits/8f338455c1b8be707350340de0368f5865ffe5a4